docs/platforms/python/tracing/span-lifecycle/index.mdx

```diff
@@ -0,0 +1,297 @@
+---
+title: Span Lifecycles
+description: "Learn how to add attributes to spans in Sentry to monitor performance and debug applications "
+sidebar_order: 40
+---
+
+<Alert>
+
+To capture transactions and spans customized to your organization's needs, you must first <PlatformLink to="/tracing/">set up tracing.</PlatformLink>
+
+</Alert>
+
+To add custom performance data to your application, you need to add custom instrumentation in the form of [spans](/concepts/key-terms/tracing/distributed-tracing/#traces-transactions-and-spans). Spans are a way to measure the time it takes for a specific action to occur. For example, you can create a span to measure the time it takes for a function to execute.
```

Comment:
```suggestion
To add custom attributes or performance data to your applications traces, you need to attach these attributes to the traces [spans](/concepts/key-terms/tracing/distributed-tracing/#traces-transactions-and-spans). Spans represent the individual actions that occur within an overall application interaction. This might include items loading, database calls, API returns, or more.  
```

This is technically inaccurate - Spans are actually the individual page actions within an existing trace. They do have a time component to them in the sense that they measure how long a thing happens; but the semantics here are that theres a lot of things that happen inside a span that we might care about (including the outputs of what a span might "do")